js/src/components/paid-ads/budget-setup/budget-setup.test.js

```diff
@@ -42,6 +47,58 @@ jest.mock( '~/hooks/useBudgetMetrics', () =>
 	} )
 );
 
+function mockRaiseBudgetRecommendationsData( { nilData } ) {
```

Comment:
```suggestion
function mockRaiseBudgetRecommendationsData( { nullData } ) {
```
Because we are more used to `null` in JS 😁 
Also, am wondering if it's more meaningful to have another function called `mockEmptyRaiseBudgetRecommendationsData`

Reply:
Updated it to `nullData`. thanks